### DISCUSSION TITLE
pdf using pdfhtml lacks page numbers

### DISCUSSION THREAD

**Discussion by actsasgeek** [executablebooks/meta#600] [Q&A]
**Background** The styling for the pdfhtml output seems much more robust than the output for pdflatex and easier to access/hack through CSS rather than the raw (la)tex especially for admonitions, etc.

However, I have struggled to find a setting that would add page numbers to the pdfhtml output and this seems like a necessity for any book.

Additionally, when using pdfhtml, I lose a title page, table of contents, and having each chapter start on a new page. Are there potential workarounds or am I stuck?

**Comment by choldgraf** [comment]
hmmm I wonder if @phaustin has guidance here - I believe he's been working on a more "book-like" PDF via HTML approach

**Reply by SimplyOm** [reply]
Hi, was there any updates on page numbering after this? I'm resorting to post-process the generated pdf using FPDF to add the page numbers.
